docs/general/ddl/data-types.rst

```diff
@@ -95,7 +95,9 @@ characters are allowed. Example::
     ... );
     CREATE OK, 1 row affected (... sec)
 
-Columns of type text can also be analyzed. See :ref:`sql_ddl_index_fulltext`.
+Columns of type text can also be analyzed. By default the
+:ref:`plain <plain-analyzer>` analyzer is used. See :ref:`sql_ddl_index_fulltext`
```

Comment:
you can break lines inside of the ref blocks if there's a space inside the backticks